ui/package.json

```diff
@@ -203,10 +203,11 @@
     "tracked-built-ins": "^3.1.1",
     "typescript": "^4.8.4",
     "walk-sync": "^2.0.2",
-    "webpack": "5.78.0",
+    "webpack": "^5.89.0",
     "xstate": "^3.3.3"
   },
   "resolutions": {
+    "@embroider/macros": "^1.13.4",
```

Comment:
Bumping this fixed an opaque build error I was getting: `node_1.MacrosConfig.transforms is not a function`.